modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/css/components/_empty_state.scss

```diff
@@ -1,6 +1,4 @@
 .taglib-empty-result-message {
-	margin-top: 82px !important;
-
```

Comment:
Hi, @rebsilva. Did you test the empty state in the journal and documents&media?

Reply:
Yes Adriano, i did the tests and everything is ok! Thanks for the remind!